src/pages/App.tsx

```diff
@@ -65,13 +64,7 @@ const App = () => (
                                 path={Path.MARKET}
                                 component={process.env.REACT_APP_RUN_V2 ? Market : MarketV1}
                               />
-                              <Route
-                                exact
-                                path={Path.MARKET_DETAILS}
-                                component={
-                                  process.env.REACT_APP_RUN_V2 ? MarketDetails : MarketDetailsV1
-                                }
-                              />
+                              <Route exact path={Path.MARKET_DETAILS} component={MarketDetails} />
```

Comment:
If this PR is to release this page and setup the version we can also remove the Market page and flag and bump the version

Reply:
We can, I wasn't sure if you were done with the Market page so I just included the Market Details page

Reply:
Done